.circleci/config.yml

```diff
@@ -137,4 +217,8 @@ workflows:
               only:
                 - master
     jobs:
-      - deploy
+      - setup_uat
```

Comment:
I see this full parallelization enables reducing time. Though I am concerned that, if master has a bug detectable by step `setup_node` or earlier step, that bug would be executed on all environments.
* Have you considering serializing most envs to earlier setup of one of the environments e.g. setup_dev1? That would slow down CI though increase confidence.
* In alternative, could you please confirm how PR gets checked before merge to master? `check` job seems performing basic sanity check though not the full setup on a dummy env.

Reply:
This is actually another feature (besides the speedup), failing one environment should not fail the others.

Most of the times the code has been run and tested manually/locally. I'm 👍 to add additional environment to run actually playbooks on it. However IMO this is not in scope of this PR.